### DISCUSSION TITLE
Import an a Geotiff as an asset using earth engine API

### DISCUSSION THREAD

**Discussion by Thandeka20** [giswqs/GEE-Courses#5] [Q&A] [answered]
Hi, 

I have a number of RGB imagery saved as tiff files, they are couple of images that cover a certain AOI. How do import my geotiffs as an image collection inorder to use use Earth Engine API on Google Colab

**Comment by giswqs** [comment, marked as answer]
See https://developers.google.com/earth-engine/guides/image_upload

**Reply by Thandeka20** [reply]
Hi, thanks for your response. But that is not what I need,  I want to upload an entire folder of images using the Python API on google co-lab. I am avoiding doing a manuel upload. But I found this link: https://github.com/google/earthengine-community/blob/master/guides/linked/Uploading_image_tiles_as_a_single_asset_using_a_manifest.ipynb which I think might be useful.
